library/src/test/java/io/constructor/core/ConstructorIoIntegrationTest.kt

```diff
@@ -77,6 +77,18 @@ class ConstructorIoIntegrationTest {
         Thread.sleep(timeBetweenTests)
     }
 
+    @Test
+    fun getAutocompleteResultsWithSectionFiltersAgainstRealResponse() {
+        val sectionFilters = mapOf(
+            "Search Suggestions" to listOf("Brand" to listOf("XYZ", "123")),
+            "Products" to listOf("availability" to listOf("ABC", "987")),
+        )
+        val observer =
+            constructorIo.getAutocompleteResults("item", sectionFacets = sectionFilters).test()
+        observer.assertComplete()
+        Thread.sleep(timeBetweenTests)
+    }
```

Comment:
Could we add assertions here to check for results? Probably can't for Search Suggestions, but we should be able to for Products.